Dolphin core downloaded from retroarch does not launch on Ubuntu 18.4

Hello,
I just downloaded last Dolphin core version from Retroarch and it doesn't launch on Ubuntu 18.04.
In my terminal I can see this error message:
```

[ERROR] Échec de l'ouverture du cœur Libretro: "/home/romain/.config/retroarch/cores/dolphin_libretro.so"
[ERROR] Error(s): /home/romain/.config/retroarch/cores/dolphin_libretro.so: symbol curl_easy_cleanup version CURL_OPENSSL_3 not defined in file libcurl.so.4 with link time reference
```

Thank you for your help.

Same issue on ArchLinux:
```
[ERROR] Impossibile aprire il core libretro : "/home/itachisan/.config/retroarch/cores/dolphin_libretro.so"
[ERROR] Error(s): /home/itachisan/.config/retroarch/cores/dolphin_libretro.so: symbol curl_easy_cleanup version CURL_OPENSSL_3 not defined in file libcurl.so.4 with link time reference
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -687,7 +687,7 @@ else()
 endif()
 
 find_package(CURL)
-if(CURL_FOUND)
+if(CURL_FOUND AND NOT(LIBRETRO))
   message(STATUS "Using shared libcurl")
   include_directories(${CURL_INCLUDE_DIRS})
 else()

